x-pack/plugins/entity_manager/server/lib/entities/built_in/containers.ts

```diff
@@ -0,0 +1,157 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EntityDefinition, entityDefinitionSchema } from '@kbn/entities-schema';
+import { BUILT_IN_ID_PREFIX } from './constants';
+
+export const builtInContainersFromEcsEntityDefinition: EntityDefinition =
```

Comment:
Should this file be called `containers` (the plural of the entity.type it detects) or `containers_from_ecs_data`? That's the (partial) id of the definition which is defined here.

Reply:
I'll rename these files